### ISSUE TITLE
Documentation : Clarify how ziglist is updated

### ISSUE THREAD

**Issue by taikedz** [pixqc/ziglist#6]
A meta-issue for sure

It is not clear how Ziglist is populated - is it done periodically? When the deno instance is relaunched ?

(It does look like it scours Github and Codeberg... somehow)

What are the criteria for inclusion?

Some notes on this in the README would be most welcome.

Thank you for Ziglist !

**Comment by pixqc** [comment]
Hi! Ziglist is updated periodically per this cron: https://github.com/pixqc/ziglist/blob/master/src/server.jsx#L721

I'm a bit behind on Ziglist lately, I'll close this issue when I've updated the readme. Thank you :)

**Comment by taikedz** [comment]
Thanks :+1: 

Can I confirm how Ziglist determines what to pick up? I do have a repo I am trying to put together for a small utility I hope to be useful ; it is public on Github with "zig" in the description, in the name, and recognised as implemented in Zig 100% per Github's language analysis, but is not showing on Ziglist.

If you prefer not to document this directly for spam avoidance, I do understand :+1:

**Comment by pixqc** [comment]
https://github.com/pixqc/ziglist/blob/master/src/main.js#L615

This is the query, it searches the title, description, etc for the keyword 'zig'. I've seen two of your Zig repo, I think both of those should've been on Ziglist but it's not search-indexed properly. Let me give it a look

**Comment by taikedz** [comment]
Yeah I saw that line - it seemed to be the only location where the search specifier is set, so I wondered whether there was a separate filtering mechanism....

Thanks :+1:

**Comment by tusharsadhwani** [comment]
ziglist might not have been updating projects for the past month or so, I think? The fix in the code seems simple, we need to add in another date in the list of dates from which it should fetch github projects, but I'm not sure how you calculate those dates exactly.

Edit: my bad, i might be mistaken.

**Comment by EzequielRamis** [comment]
Hey @pixqc! I'm not seeing [zimdjson](https://github.com/EzequielRamis/zimdjson) indexed, which has all the queryable info, so there may be a bug.

**Comment by Tony-ArtZ** [comment]
hey same issue here, just going to bump this up. not sure if it has been fixed but there is this  [package](https://github.com/Tony-ArtZ/zorm.git) I made, might be helpful to someone

**Comment by pixqc** [comment]
Hello and thank you for notifying me. This is definitely an issue on my end, I'll be giving the indexing issue a look when I do have free time, my apologies!
